docs/pages/billing/plans.mdx

```diff
@@ -0,0 +1,94 @@
+  src="/static/images/billing/plan-01.png"
+  style={{ maxWidth: 720 }}
+/>
+
+You can also cancel a subscription at any time. See [Cancel a plan](/billing/manage/#cancel-a-plan) for more information.
+
+We also offer annual contracts on an as-needed basis. Contact our [customer support](https://expo.dev/contact) team to see if an annual plan suits you.
+
+## Plans
+
+Each plan has specific limits. However, subscribers can exceed them and pay for additional usage with [usage-based billing](/billing/usage-based-pricing/).
+
+<BoxLink
+  title="Pricing"
+  description="Visit our pricing page to see the list of current prices for each plan."
+  href="https://expo.dev/pricing"
+  Icon={CreditCard02Icon}
+/>
+
+### Production
+
+The Production plan is designed for professional developers and small businesses. Subscribing to this plan gets you access to:
+
+- Reliable, production-grade services
+- Monthly [credits](/billing/usage-based-pricing/) for high-priority builds for EAS Build
+- More unique users, higher bandwidth, and storage limit for EAS Update
+
+If you exceed the limits or use up your monthly credits, any further usage will be charged at [usage-based prices](/billing/usage-based-pricing/).
+
+### Enterprise
+
+The Enterprise plan is designed for organizations and enterprises that have large projects and require additional resources such as dedicated support. Subscribing to this plan gets you access to:
+
+- Reliable, production-grade services
+- Monthly [credits](/billing/usage-based-pricing/) for high-priority builds for EAS Build
+- More unique users, higher bandwidth, and storage limit for EAS Update
+
+An Enterprise plan offers much higher monthly credits for EAS Build and bandwidth for EAS Update than any other plan.
+
+If you exceed the limits or use up your monthly credits, any further usage will be charged at [usage-based prices](/billing/usage-based-pricing/).
+
+### On-demand
+
+The On-demand plan is a pay-as-you-go option for developers who have variable usage of our services. You can dial up or down your usage as you see fit and upgrade to the Production or Enterprise plan if needed.
+
+Subscribing to the On-demand plan gets you access to:
+
+- Reliable, production-grade services
+- High-priority builds with EAS Build at usage-based prices
+- The ability to exceed the limits of the Free plan for EAS Update and pay for additional usage
+
+## Usage-based billing
+
+Usage-based billing is applied to customers who exceed their plan limits. It enables you to use our services without worrying about limitations or any contractual obligations.
+
+Usage-based billing is billed monthly and is currently enabled for EAS Build and EAS Update. We provide an estimate of your existing usage and any overage charges on [your account's Billing](https://expo.dev/settings/billing).
+
+## Add-ons
+
+### Enterprise Support
+
+The Enterprise Support add-on is only available for Enterprise plan subscribers. Key features include:
```

Comment:
May want to set expectations that support add-on is available to new customers if we have enough capacity.

Reply:
I've added the context here to slightly rephrase it to: "The Enterprise Support add-on is only available for new Enterprise plan subscribers, subject to availability."